### PR TITLE
Don't import package files in setup

### PR DESCRIPTION
**Pull request by vilyapilya**
setup should not import the files from the package. Open and read the version file.

### CODE REVIEW

**Comment by smusali** [comment]
> What's the advantage of having that file instead of just defining a constant in this file?

This is a very interesting question, by the way - in `Node.js` stuff, we handle everything inside `package.json` without having any extra file containing the version - here `setup.py` has the same mission - my answer to @dchai76's question is there is no significant difference - it might be good for just cleanly updating the version but since our `setup.py` is small and easily manageable, I think we can just remove the file containing version information.

Another thing is it's getting used in `configs.py` for defining `user-agent` but still `__version__` can be exported and used from `setup.py` in the same way.

**Comment by dchai76** [comment]
If you do want to keep the file, I'd just call it something like `VERSION` - doesn't need to be a `.py` file.

**Comment by vilyapilya** [comment]
@dchai76  I am using the variable of this file in the code to extract the version for user-agent. The advantage is that we have a single place where the version is defined.

File: setup.py [hunk 1]
```diff
@@ -3,21 +3,23 @@
...
   packages = ['logdna'],
-  version = _version.__version__,
+  version = version,
   description = 'A Python Package for Sending Logs to LogDNA',
   author = 'LogDNA Inc.',
   author_email = 'help@logdna.com',
   license = 'MIT',
   url = 'https://github.com/logdna/python',
-  download_url = ('https://github.com/logdna/python/tarball/%s' %(_version.__version__)),
+  download_url = ('https://github.com/logdna/python/tarball/%s' %(version)),
```
Comment (dchai76) [inline setup.py]: You don't need the parentheses around `version`, and a space after `%` is more typical formatting, but OK.

**Pull request merged**